README.md

```diff
@@ -1 +1,115 @@
-# Prefect Collection Registry
\ No newline at end of file
+# Prefect Collection Registry
+This is a registry of metadata for Prefect collections.
+
+<p align="center">
+    <img src=imgs/registry.png width=400>
+</p>
+
+## What metadata is stored here?
+This repository stores metadata for:
+- the Collections themselves
+- the flows defined in those collections
+- the blocks defined in those collections
+
+## Repository Structure
+This repository is structured as follows:
+
+```
+prefect-collection-registry/
+|-- collections/
+|   |-- collection-A/
+|   |   |-- latest-release-tag-n.json
+|   |-- ...
+|-- flows/
+|   |-- collection-A/
+|   |   |-- release-tag-n.json
+|   |   |-- ...
+|   |   |-- release-tag-0.json
+|   |-- ...
+|-- blocks/
+|   |-- collection-A/
+|   |   |-- release-tag-n.json
+|   |   |-- ...
+|   |   |-- release-tag-0.json
+|   |-- ...
```

Comment:
We discussed using a repo structure like this:
```
prefect-collection-registry/
|-- collections/
|   |-- collection-A/
|   |   |-- latest-release-tag-n.json
|   |   |-- flows/
|   |   |   |-- release-tag-n.json
|   |   |   |-- ...
|   |   |   |-- release-tag-0.json
|   |   |-- blocks/
|   |   |   |-- release-tag-n.json
|   |   |   |-- ...
|   |   |   |-- release-tag-0.json
|   |-- ...
```
Should we update this PR to display that structure in the docs?